### ISSUE TITLE
JAWS doesn't read aria-label for menu groups

### ISSUE THREAD

**Issue by Leuvarden** [FreedomScientific/VFO-standards-support#381]
### Summary

> 
> Example:
> 1. Go to https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-2/menubar-2.html
> 2. Open Style/Color submenu
> 3. Navigate with Jaws through menu items

### Expected result

> Aria-labels for <ul role="group"> should be read to announce information about each type of menuitems

### Actual result

> Aria-labels are not read, only labels of menuitems 

### Additional Information
Speech History: https://i.imgur.com/qTsjw76.png

#### JAWS version and build number
JAWS 18.0.5038 and JAWS 2018.1811.30

#### Operating System and version
Windows 10 Enterprise

#### Browser and version:
Chrome 79.0, Firefox 72.0

**Comment by ferllings** [comment]
Hello @Leuvarden,

I don't see any aria-label for the menuitems. Could you describe what output do you expect exactly?

**Comment by ferllings** [comment]
I see what you mean now.

I can reproduce on Chrome 81 + Jaws 2020

**Comment by adelara** [comment]
Would you mind explaining the "I see what you mean"  as I'm not getting the issue. I'm on Chrome 83  and Jaws 2020. Thanks.

**Comment by scottaohara** [comment]
the specific markup example in the original post was not surrounded by markdown tic-marks to have it properly render.  

the issue is that there are instances of `ul` with `role=group` and `aria-label=name-of-group` in the APG example that JAWS was not reading at the time of this ticket being created, and were still an issue as of June 8th.

The expectation is that if authors are properly grouping menuitems within a group, which is a valid child of the `role=menu` then the group title should be announced by JAWS when a user navigates into the new grouping of menuitems.

Retesting with Chrome 83 and JAWS 2020.2006.12 with the [APG 1.2 pattern](https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-2/menubar-2.html) (which fixes some markup issues with the 1.1 pattern) the group labels are not announced when navigating between the exposed menu groupings.

@BenKeyFSI

**Comment by smhigley** [comment]
Are there any updates on this? I still see the issue currently with JAWS 2022. Here's a very simplified repro for testing if it helps: https://jsfiddle.net/ac7guekz/
